neutrons/reflectivity_ui
# Integration test cross section with no proton charge

### Description of work:


### To Test by Reviewer:
Checkout the PR and run UI tests. It's necessary that `/SNS/REF_M` is mounted on the testing machine
```bash
git fetch origin pull/55/merge:pr55
git switch pr55
conda activate quicknxs  # see conda environment file "environment.yml"
pytest ./test/ui
```
The reviewer should check for the discussed software practices

- [ ] all internal functions should have an underbar, as is python standard
- [ ] need more comments in meat of code explaining intent
- [ ] does the code follow the other [coding standards](https://scse.ornl.gov/drtsans/developer/index.html)?
